nixos/modules/services/audio/upmpdcli.nix

```diff
@@ -0,0 +1,88 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+with pkgs;
+with lib;
```

Comment:
Please avoid file-wide `with` statements, either make a list of `inherit`s up top or just prefix usage with `pkgs.` and `lib.`.